client/js/lounge.js

```diff
@@ -194,19 +195,27 @@ $(function() {
 		input.val("");
 		resetInputHeight(input.get(0));
 
-		if (text.indexOf("/clear") === 0) {
-			utils.clear();
-			return;
-		}
-
-		if (text.indexOf("/collapse") === 0) {
-			$(".chan.active .toggle-button.opened").click();
-			return;
-		}
-
-		if (text.indexOf("/expand") === 0) {
-			$(".chan.active .toggle-button:not(.opened)").click();
-			return;
+		if (text.indexOf("/") === 0) {
+			const separatorPos = text.indexOf(" ");
+			const cmd = text.substring(1, separatorPos > 1 ? separatorPos : text.length);
+			const parameters = separatorPos > text.indexOf(cmd) ? text.substring(text.indexOf(cmd) + cmd.length + 1, text.length) : "";
+			switch (cmd) {
```

Comment:
This can be simplified like `if (cmd in modules)` or `if (typeof modules[cmd] === "function")`